net/miniupnpd/patches/100-clearer-protocol-labels.patch

```diff
@@ -0,0 +1,75 @@
+From bbed78899e275d5b89774cb8e14e3a3ce0c14484 Mon Sep 17 00:00:00 2001
+From: Self-Hosting-Group
+ <155233284+Self-Hosting-Group@users.noreply.github.com>
+Date: Mon, 12 May 2025 00:00:00 +0000
+Subject: [PATCH] miniupnpd: Clearer / less redundant protocol labels in
+ description
```

Comment:
Why do we need this patch? Why it is useful to have this merged?
It is not clear for me. I doubt that something like this is accepted by miniupnpd repository.